job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/ExecuteDslScripts.java

```diff
@@ -245,14 +268,28 @@ public String getAdditionalClasspath() {
 
     @DataBoundSetter
     public void setAdditionalClasspath(String additionalClasspath) {
-        this.additionalClasspath = additionalClasspath;
+        this.additionalClasspath = StringUtils.isBlank(additionalClasspath) ? null : additionalClasspath;
```

Comment:
Consider using [hudson.Util.fixEmpty(string)](http://javadoc.jenkins-ci.org/hudson/Util.html#fixEmpty-java.lang.String-) for the exact same behavior.